scripts/plant_growth_form.py

```diff
@@ -0,0 +1,73 @@
+#retriever
+from retriever.lib.templates import Script
+from retriever.lib.models import Table, Cleanup, correct_invalid_value
+from pkg_resources import parse_version
+try:
+    from retriever.lib.defaults import VERSION
+    try:
+      from retriever.lib.tools import open_fr, open_fw, to_str
+    except ImportError:
+      from retriever.lib.scripts import open_fr, open_fw
+except ImportError:
+    from retriever import open_fr, open_fw, VERSION
+
+class main(Script):
+    def __init__(self, **kwargs):
+        Script.__init__(self, **kwargs)
+        self.name = "plant-growth-form"
+        self.ref = "http://onlinelibrary.wiley.com/doi/10.1002/ecy.1569/full"
+        self.urls = {"plantgrowth": "http://onlinelibrary.wiley.com/store/10.1002/ecy.1569/asset/supinfo/ecy1569-sup-0001-DataS1.zip?v=1&s=219c74fa85bfe6738e649bedd844f40428351802"}
+        self.citation = "Engemann, K., Sandel, B., Boyle, B. L., Enquist, B. J., Jorgensen, P. M., Kattge, J., McGill, B. J., Morueta-Holme, N., Peet, R. K., Spencer, N. J., Violle, C., Wiser, S. K. and Svenning, J. .-C. (2016), A plant growth form dataset for the New World. Ecology. Accepted Author Manuscript. doi:10.1002/ecy.1569"
+        self.tags = ['Taxon > Plants', 'Data Type > Observational']
+        self.retriever_minimum_version = "2.0.dev"
+        self.script_version = 1.0
+        self.description = "This dataset provides growth form classifications for 67,413 vascular plant species from North, Central, and South America."
+
```

Comment:
let us add the code below here. 
[why]: So we try to make these scripts compatible with the old version of the retriever. if someone is using retriever version `2.0.dev`, they could be able to get this new script and use it.

```        if parse_version(VERSION) <= parse_version("2.0.0"):
            self.shortname = self.name
            self.name = self.title
            self.tags = self.keywords
```